app.js

```diff
@@ -88,6 +75,7 @@ class ToDoList {
             if (taskList[i].id == idOfTaskToDelete) {
```

Comment:
To samo co w changeStatus, zmien `for` na `.find`. https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/find